routes/submitForm.js

```diff
@@ -1,16 +1,63 @@
-/*************************
-TODO PART 1
-TODO find a module that can submit mysql requests
-TODO write a query request in the function below
-TODO make it transaction based
-
-TODO Part2
-TODO put the config info for the mysql server in the config
-TODO send the front end a success message on success and an error message with error code on fail
-TODO remove these comments!
-*************************/
+var mysql = require("mysql");
+var config = require("konfig")();
 
 var router = function(req, res) {
+    var connection = mysql.createConnection({
+        host: config.app.mysql.host,
+        user: config.app.mysql.user,
+        password: config.app.mysql.password
+    });
+
+    connection.connect(function(err) {
+        if (err) {
+            console.error(err.stack);
+            res.send("Connection Error: " + err.code);
```

Comment:
When sending errors it is good to specify an error code:
res.status(503).send(...)

This lets the front end display it as an error rather than a success. Status codes can be found here: http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html